app/views/collections/_collection_description.erb

```diff
@@ -1,5 +1,5 @@
 <% presenter.description.each do |description| %>
     <p class="collection_description">
-        <%= iconify_auto_link(format_description(description)) %>
```

Comment:
No longer needed here -- we're treating both types of description the same way, which includes a pass through iconify_auto_link in the description formatter above.